### DISCUSSION TITLE
USB cam plugged once in a QIDI x-Plus 3. video: OK --- disconnected and reconnected: KO

### DISCUSSION THREAD

**Discussion by shottydan** [fluidd-core/fluidd#1415] [Q&A] [answered]
as title, connected USB cam in a QIDI X-Plus 3 for Fluidd webpage printer video control, only saved a name for cam, no modifications to oher settings: video was perfect (also through PC USB port)
cam disconnected, reconnected after one hour, Fluidd no more recognize same cam, no more, also after tenth of different settings in Fluidd camera section.

hey guys, any idea about this weird issue and how to resolve?

P.S.: USB cam re-checked, and is ok in W11, as a driverless...
P.S.2: plugged at first in pendrive printer usb portand video OK, e-connected, no more video, even if plugged directly in QIDI mboard USB free port.
P.S:3: checked  QIDI pendrive port and cam in PC: both are OK

**Comment by pedrolamas** [comment]
What version of Fluidd are you using?

**Comment by shottydan** [comment]
Hi!!
thanks a lot for fast reply 💪

I don't know which is the version, I'm using the Fluidd webpage, appeared
when I put printer IP x.x.x.x:10088 in Chrome plus.
as I wrote, the odd part is that at first connection, at pendrive port on
the top of printer, all was ok and when reconnected: nothing... 🙄

thanks again and have a nice day

;-Dan from Italy

[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
14/04/24,
10:25:55

Il giorno sab 13 apr 2024 alle ore 21:15 Pedro Lamas <
***@***.***> ha scritto:

**Reply by pedrolamas** [reply]
Fluidd version is shown on the bottom right corner of the webpage.

**Comment by shottydan** [comment]
sorry...
[image: image.png]

thanks again for help

[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
14/04/24,
12:38:29

Il giorno dom 14 apr 2024 alle ore 10:33 Pedro Lamas <
***@***.***> ha scritto:

**Reply by pedrolamas** [reply]
There is no image here. Please reply directly via browser, not by replying to the email!

**Reply by shottydan** [reply]
sorry! I'm new on this...

![image](https://github.com/fluidd-core/fluidd/assets/34281529/a16e6146-9780-4ddb-9f06-83a5b59202b7)

**Reply by shottydan** [reply]
here in text

@ 2024 fluidd: v1.19.0-6184c73

**Reply by pedrolamas** [reply, marked as answer]
Fluidd v1.19.0 was released 2 years ago, so that version is really old!

You should update Fluidd to the most recent version by going to Fluidd Settings, scrolling down to the "Software Updates" and updating from there.

However, be aware that I have heard other users complaining that they cannot see the "Software Updates" section in their Qidi printers, for which it seems Qidi disabled it.

In that case, your best option is to go to Qidi for support as we do not support such old versions of Fluidd anymore.

**Comment by shottydan** [comment]
I read that I have to install moonraker before updating, even if I've no idea how to do this...
I'm not able because never used this stuffs, but if you say that is QIDI to block it, and installing moonraker don't solve the problem, I'll go there for some help.
that is difficult to understand is why at first all was right and now no more...

**Reply by shottydan** [reply]
I'm trying to install fluidd from scratch, but github stuffs aren't in my skills. I'm good in Windows. may be to reinstall Fluidd a valid attempt to try solving my problem or not?
I approached to GitHub with Marlin configuration, but with a guided installation, and not understood a lot about this

**Reply by pedrolamas** [reply]
Be advised that Qidi has made specific changes to Klipper, Moonraker, and even Fluidd, and things will probably stop working correctly if you try to update directly!

My advice is that you contact Qidi support directly to avoid any major problems with your printer.
